packages/gestalt/src/RadioButton.css

```diff
@@ -1,4 +1,8 @@
 :root {
+  --border-sm: 6px;
```

Comment:
@AlbertCarreras could we also add the hover state? We probably want to set this on the `.input` element.

![image](https://user-images.githubusercontent.com/127199/77101624-d8d45a00-69d4-11ea-98ba-f8ea9918fe1c.png)

Reply:
You mean a darker gray when hovered?

Reply:
DONE